addon/components/keyboard-navigable-list.js

```diff
@@ -31,11 +31,17 @@ export default Ember.Component.extend(EKMixin, {
   click(e){
     // if a user clicks on an item, we want to intercept the click
     // and update the activeItem with the item that was clicked on.
-    let $e = Ember.$(e.target);
+    let target = e.target;
+
+    // if the target is an anchor tag we need to get it's parent li
+    if(target.nodeName.toUpperCase()==='A'){
```

Comment:
we need to check if this is the right node (has the data-item-index) if it is we use it, if not we check the parent. Just checking an anchor is too fragile.